install.sh

```diff
@@ -783,4 +793,17 @@ openssl_get() {
         openssl s_client -quiet -connect "$host:443" 2>/dev/null
 }
 
+lines_num(){
+    echo "$1" | wc -l
+}
+
+is_multiline(){
```

Comment:
As per my other comment, this function could be reduce to just something like:

```
test $(echo -n "$1" | wc -l) -gt 1
```

Reply:
Would you prefer having the `is_multiline` method in there for future use or should I test for multiline directly in the config check body?

Reply:
I think having `is_multiline` adds to readability of the code but it's your call @rs

Reply:
Yes keep the function for readability.